core/src/main/java/com/digitalpebble/behemoth/BehemothDocument.java

```diff
@@ -68,6 +65,8 @@ public BehemothDocument() {
     /** List holding the annotations **/
     private List<Annotation> annotations;
 
+    public BehemothDocument() {}
```

Comment:
That's a formatting matter, I'd rather leave it as is for consistency across the project. If we decide to change the format then we'll do that via the formatting rules and across the whole project.